game/src/info/trip.rs

```diff
@@ -496,26 +496,19 @@ fn make_timeline(
             (100.0 * percent_duration) as usize
         )));
 
-        // TODO We're manually mixing screenspace_svg, our own geometry, and a centered_on(). Be
```

Comment:
This is exactly the sort of thing I'm chuffed to see just work on both our screens after this PR. I sunk way too much time trying to get this old cruft to work. :)